spec/contracts.dd

```diff
@@ -99,18 +97,22 @@ body
     return cast(long)std.math.sqrt(cast(real)x);
```

Comment:
Maybe @MetaLang wants to update contracts.html and grammar.html? Anyway, AIUI the current stable compiler doesn't support it yet.